templates/StatefulSet-zabbix-proxy.yaml

```diff
@@ -25,6 +25,24 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-proxy
         app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-proxy
     spec:
+      nodeSelector:
```

Comment:
Hi @adamcharnock!

Could you remove the lines 28 to 33?
The goal is to leave only the lines 34 to 45 that will be filled in only when someone sets the custom values for these properties.